etl/steps/data/garden/irena/2022-10-20/renewable_power_generation_costs.meta.yml

```diff
@@ -0,0 +1,47 @@
+dataset:
+  namespace: irena
+  title: Renewable power generation costs
+  short_name: renewable_power_generation_costs
+  publication_year: 2022
+  publication_date: 2022-07-01
+  version: 2022-10-20
+  description: |
+    Levelized cost of energy (LCOE) estimates the average cost per unit of energy generated across the lifetime of a new power plant. It is measured in 2021 US$ per kilowatt-hour.
+  sources:
+    -
+      name: International Renewable Energy Agency (IRENA)
+      published_by: © 2022 by International Renewable Energy Agency (IRENA)
+      url: https://irena.org/publications/2022/Jul/Renewable-Power-Generation-Costs-in-2021
+      date_accessed: 2022-10-20
+
+tables:
+  renewable_power_generation_costs:
+    variables:
+      bioenergy:
+        title: Bioenergy levelized cost of energy
+        short_unit: $/kWh
+        unit: 2021 US$ per kilowatt-hour
```

Comment:
It's not very clear what the 2021 means here and it looks a little odd in the hover on maps. Maybe better to have this information in the variable description.